parlai/crowdsourcing/tasks/acute_eval/example_script.py

```diff
@@ -3,97 +3,70 @@
-    "--subtasks-per-unit",
-    default=5,
-    help="Number of conversations to evaluate per task, default 5",
-    type=int,
-)
-
-architect_type, requester_name, db, args = parser.parse_launch_arguments()
-
-USE_LOCAL = True
-
-task_title = "Which Conversational Partner is Better?"
-task_description = "Evaluate quality of conversations through comparison."
-hit_keywords = "chat,evaluation,comparison,conversation"
-
-ARG_STRING = (
-    f"--blueprint-type {BLUEPRINT_TYPE} "
-    f"--architect-type {architect_type} "
-    f"--requester-name {requester_name} "
-    f'--task-title "\\"{task_title}\\"" '
-    f'--task-description "\\"{task_description}\\"" '
-    "--task-reward 0.5 "
-    f"--task-tags {hit_keywords} "
-    f"--maximum-units-per-worker 0 "  # Num of units a worker is allowed to do, 0 is infinite
-    f"--allowed-concurrent 1 "  # Workers can only do one task at a time, or onboarding may break
-)
-
-extra_args = {
-    "pairings_filepath": args['pairings_filepath'],
-    "block_on_onboarding_fail": True,
-    "block_qualification": f"acute_eval_{int(time.time())}_block",
-    # num times to use the same conversation pair
-    "annotations_per_pair": args["annotations_per_pair"],
-    "random_seed": 42,  # random seed
-    "subtasks_per_unit": args[
-        "subtasks_per_unit"
-    ],  # num comparisons to show within one unit
-    "num_matchup_pairs": args[
-        "num_matchup_pairs"
-    ],  # num pairs of conversations to be compared
-    # question phrasing
-    "s1_choice": "I would prefer to talk to <Speaker 1>",
-    "s2_choice": "I would prefer to talk to <Speaker 2>",
-    "eval_question": "Who would you prefer to talk to for a long conversation?",
-    "assignment_duration_in_seconds": 600,
-}
-
-operator = Operator(db)
-operator.parse_and_launch_run_wrapper(shlex.split(ARG_STRING), extra_args=extra_args)
-operator.wait_for_runs_then_shutdown(skip_input=True, log_rate=30)
+TASK_DIRECTORY = os.path.dirname(os.path.abspath(__file__))
+
+defaults = [
+    {"mephisto/blueprint": BLUEPRINT_TYPE},
+    {"mephisto/architect": "local"},
+    {"mephisto/provider": "mock"},
+    {"conf": "example"},
+]
+
+
+@dataclass
+class TestScriptConfig(RunScriptConfig):
+    defaults: List[Any] = field(default_factory=lambda: defaults)
+    task_dir: str = TASK_DIRECTORY
+    current_time: int = int(time.time())
+
+
+register_script_config(name='scriptconfig', module=TestScriptConfig)
+
+
+@hydra.main(config_name="scriptconfig")
+def main(cfg: DictConfig) -> None:
+    db, cfg = load_db_and_process_config(cfg)
+
+    world_opt = {}
+
+    custom_bundle_path = cfg.mephisto.blueprint.get("custom_source_bundle", None)
+    if custom_bundle_path is not None:
+        assert os.path.exists(custom_bundle_path), (
+            "Must build the custom bundle with `npm install; npm run dev` from within "
+            f"the {TASK_DIRECTORY}/webapp directory in order to demo a custom bundle "
+        )
+        world_opt["send_task_data"] = True
```

Comment:
This isn't relevant for your task, was just for ParlAI custom tasks

Reply:
Oy, yes, good catch